networkx/algorithms/centrality/tests/test_harmonic_centrality.py

```diff
@@ -92,3 +94,21 @@ def test_singleton(self):
         c = harmonic_centrality(G, distance="weight")
         d = {0: 0}
         assert c == d
+
+    def test_cycle_c4_directed(self):
+        c = harmonic_centrality(self.C4_directed, nbunch=[0, 1], sources=[1, 2])
+        d = {0: 0.833, 1: 0.333}
+        for n in sorted([0, 1]):
```

Comment:
```suggestion
        for n in [0, 1]:
```
Just a nit - sorted not necessary here.